core/commands/files/files.go

```diff
@@ -397,8 +397,8 @@ Examples:
 		cmds.StringArg("path", true, false, "Path to file to be read."),
 	},
 	Options: []cmds.Option{
-		cmds.IntOption("offset", "o", "Byte offset to begin reading from."),
-		cmds.IntOption("count", "n", "Maximum number of bytes to read."),
+		cmds.IntOption("n", "count", "Maximum number of bytes to read.").Default(-1),
+		cmds.IntOption("o", "offset", "Byte offset to begin reading from.").Default(0),
 	},
```

Comment:
Why did you change it to make the short run first?  Didn't @Kubuxu say: "AFAIK we settled on the first option name being the primary one/full".